tests/shell/shell_recorder/globalplugins.esr

```diff
@@ -1,32 +0,0 @@
-Mountpoint: system/
-DiffType: File
-
-
-< set system/elektra/globalplugins ""
```

Comment:
Isn't here only `kdb` missing?

The tests worked some time ago, the `kdb global-mount` tool was implemented according this schema.

@tom-wa Do you have an idea what is not working here?